.gitignore

```diff
@@ -82,7 +82,8 @@ hs_err_pid*.log
 /RecentActivity/release/
 /CentralRepository/release/
 
-/.idea/
+.idea/
+*.iml
```

Comment:
I am unsure what this change to the gitignore is fixing / why it is being modified is there are comment about why this is being changed somewhere that i missed? (as I have currently only done an initial skimming of all the comments)